nca/NetworkConfig/NetworkConfigQuery.py

```diff
@@ -2255,13 +2255,13 @@ def check_intersects_original(self, only_captured=True):
 
     def check_intersects_optimized(self, only_captured=True):
         if only_captured:
-            res_conns_filter1 = OptimizedPolicyConnections(make_allowed=True, make_denied=False,
-                                                           make_pass=False, make_all_allowed=False)
+            res_conns_filter1 = PolicyConnectionsFilter(calc_allowed=True, calc_denied=False,
+                                                        calc_pass=False, calc_all_allowed=False)
         else:
-            res_conns_filter1 = OptimizedPolicyConnections(make_allowed=False, make_denied=False,
-                                                           make_pass=False, make_all_allowed=True)
-        res_conns_filter2 = OptimizedPolicyConnections(make_allowed=False, make_denied=False,
-                                                       make_pass=False, make_all_allowed=True)
+            res_conns_filter1 = PolicyConnectionsFilter(calc_allowed=False, calc_denied=False,
+                                                        calc_pass=False, calc_all_allowed=True)
+        res_conns_filter2 = PolicyConnectionsFilter(calc_allowed=False, calc_denied=False,
+                                                    calc_pass=False, calc_all_allowed=True)
```

Comment:
suggest having functions to simplify this, such as:
```
res_conns_filter1 = only_allowed_connections()
```
with func:
```
def only_allowed_connections:
  return PolicyConnectionsFilter(calc_allowed=True, calc_denied=False,
                                                        calc_pass=False, calc_all_allowed=False)
```

Reply:
Done